### ISSUE TITLE
In Readme it should be 'Config::Integrations::Rails::Railtie.preload'

### ISSUE THREAD

**Issue by altmer** [railsconfig/config#145]
Currently in readme it says that in order to use config in initializers you should use code

```
Config::Integration::Rails::Railtie.preload
```

when actually it should be (with -s)

```
Config::Integrations::Rails::Railtie.preload
```

**Comment by pkuczynski** [comment]
Fixed in #146 by @kris

**Comment by pkuczynski** [comment]
@altmer, using `preload` mentioned in the README should not be required anymore, as Config should be loaded before Rails configuration. Example basic Rails app illustrating this case is presented on this repo: https://github.com/railsconfig/config-rails-4.2

I am writing to you, as I assume you used the README suggestion in you application. Could you please remove it from your app and check it that's still working? I assume it should and then I could remove this unnecessary description from README...
